tests/phpunit/MessageReporterAwareTest.php

```diff
@@ -1,41 +0,0 @@
-<?php
```

Comment:
Why is this removed?

Reply:
Because the test is not testing any production code logic and violates PSR-1 by having two classes in a file

Reply:
> Because the test is not testing any production code logic and violates PSR-1 by having two classes in a file

Well, then is needs to be an integration test. The purpose of this exercise is that `MessageReporterAwareTrait` is tested to be working as expected (it should but tests provide clarity on the intent and behaviour).

Reply:
Oh... I had not seen the trait. Probably due to a combination of me not being used to traits and always placing them on the first line in the class, not the second one.